### ISSUE TITLE
Possible false-positive for vue-a11y/click-events-have-key-events

### ISSUE THREAD

**Issue by Stegosource** [vue-a11y/eslint-plugin-vuejs-accessibility#10]
The `vue-a11y/click-events-have-key-events` complains whenever an `@click` handler is placed on a button without also including `@keydown.enter` and `@keyup.space`.

However, adding these listeners to a button, then pressing the `enter` or `space` keys will trigger the click event as well, thus firing the event twice.

This has been documented and there is a slightly hacky workaround here: https://github.com/maranran/eslint-plugin-vue-a11y/issues/28

As I understand it, this rule is actually intended for non-interactive elements. So, for example:
```
<div @click="handler">
```
Should actually be:
```
<div @click="handler" @keydown.enter="handler" @keydown.space="handler" tabindex="0">
```

As I understand, this is not necessary for the button element. Just add the click handler, and the keybaord events for `enter` and `space` will fire the click handler as well.

If my understanding is incorrect, please let me know. I'd like to read more about that. If my understanding is correct, then we may want to add some conditional logic to only show this error for non-interactive elements.

**Comment by kddeisz** [comment]
I'm going to close this assuming since on twitter you said you got this resolved. Let me know if you have further issues!

**Comment by braddialpad** [comment]
Can we know what the fix was?

**Comment by AustinGil** [comment]
Hey @braddialpad, I looked through my projects but did not find where the problem/solution was. 

Maybe if you share your code it will jog my memory.
